wiki/markdown/configuration.md

```diff
@@ -168,4 +168,57 @@ and the settings are defined in the **ldap.server.list** property.
             "password": "",\
             "user.filter": "uid={0}"\
         }\
-    ]\
\ No newline at end of file
+    ]\
+
+## Password encryption
+
+In a production environment and filesystems, administrator most likely don't want to store passwords in clear text.
+
+These Eiffel-Intelligence password properties can be provided encrypted:
+        
+        rabbitmq.password
+        spring.data.mongodb.password
+
+
+A password property is configured by providing the encrypted password in this format.
+
+        some.component.password=ENC(<encrypted password>)
+
+
+Ldap encrypted password is set inside **ldap.server.list** property according the example below:
+ 
+    ldap.server.list=[{\
+        "url": "<ldap address>",\
+        "base.dn": "<dn>",\
+        "username": "<username>",\
+        "password": "ENC(<encrypted password>)",\
+        "user.filter": "<filter>"\
+    }]
+
+How and examples of how to encrypt passwords can be found on Jasypt homepage.
```

Comment:
Remove the first "How and" in the beginning of the sentence.

Reply:
Done.